content/concept-amendments.md

```diff
@@ -179,33 +182,54 @@ An address with a SignerList can disable the master key even if a regular key is
 * `tefNOT_MULTI_SIGNING`
 * `tefBAD_AUTH_MASTER`
 
-<!--{# to be added in a later version:
+
 ## OwnerPaysFee
 
-| Amendment ID | Status |
-|--------------|--------|
-| TBD | Coming soon |
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 9178256A980A86CF3D70D0260A7DA6402AAFE43632FDBCB88037978404188871 | In development. |
+
+Fixes an inconsistency in the way [transfer fees](concept-transfer-fees.html) are calculated between [OfferCreate](reference-transaction-format.html#offercreate) and [Payment](reference-transaction-format.html#payment) transaction types. Without this amendment, the holder of the issuances pays the transfer fee if an offer is executed in offer placement, but the initial sender of a transaction pays the transfer fees for offers that are executed as part of payment processing. With OwnerPaysFee, the holder of the issuances always pays the transfer fee, regardless of whether the offer is executed as part of a Payment or an OfferCreate transaction. Offer processing outside of payments is unaffected.
+
+This Amendment requires the [Flow Amendment](#flow) to be enabled.
+
+
+## PayChan ##
+
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 08DE7D96082187F6E6578530258C77FAABABE4C20474BDB82F04B021F1A68647 | In development. |
+
+Creates "Payment Channels" for XRP. Payment channels are a tool for facilitating repeated unidirectional payments or extensions of credit between two parties. Ripple expects this feature to be useful for the [Interledger Protocol](https://interledger.org/). One party creates a Payment Channel and sets aside some XRP in that channel for a predetermined expiration. Then, through off-ledger secure communications, the sender can send "Claim" messages to the receiver. The receiver can redeem the Claim messages before the expiration, or choose not to in case the payment is not needed. The receiver can verify Claims individually without actually distributing them to the network and waiting for the consensus process to redeem them, then redeem the batched content of many small Claims later, as long as it is within the expiration.
```

Comment:
Yeah, that's what I meant. But maybe it should be reworded so it can't be confused with RCL trust lines...